K8's removing pages and links (#354)
tigergraph/server-docs: modules/kubernetes/pages/index.adoc

> Previous versions of TigerGraph used a different implementation of Kubernetes. This is still functional, but no longer supported. For long-term support and enhancement, use the Operator instead.

Reviewer: ```suggestion
Previous versions of TigerGraph used a different implementation of Kubernetes.
The older versions may be still functional, but no longer supported. For long-term support and enhancement, use the Operator instead.
```
Ambiguous what is "this".
Also, because this changeover took place a while ago now, I'm not positive that the older versions still work with the newer database.

Author: Thank you. this was changed in the latest commit.
